examples/8a_benchmark_write_parallel.cpp

```diff
@@ -387,6 +389,11 @@ void parse(TestInput& input, std::string line)
     return;
   }
 
+  if ( vec[0].compare("prefix") == 0 ) {
```

Comment:
First, would be good to call it something like "DefaultFileLocation", to spell out the purpose.

Second, if a user provided such a path,  shall we first check whether it is a valid one with right permissions, If not, ask the user to create it first.

Reply:
For the second part: `openPMD::Series()` will create similar to `mkdir -p` directories that are passed and not yet existing.

Permission issues would be forwarded through it (can maybe be improved generally, if they should not propagate well).

Reply:
I've renamed the variable to `fileLocation` to make it clearer. Thanks for the feedback! I leave it up to you to decide the expected behavior: (1) we check and ask the user to create if it is not there or (2) we transparently create the directory and continue the execution. What would be the best?

Reply:
Yes, I know Series() creates recursively. 

I would like to give extra precaution when doing so for user provided path.   

Guess I will leave it to Axel to decide on.  @ax3l

Reply:
@ax3l what is your take on this?

Reply:
Let's leave the default behavior: user provides a path, we will create it if it does not exist.